app/assets/stylesheets/common/_core.scss

```diff
@@ -78,6 +77,15 @@ label.required::after {
   }
 }
 
+#loggedIn {
+  height: 100%;
+}
+
+#wrapper {
+  background: $white;
```

Comment:
This changes the background color of the nav bar, which should remain the colour it was before.